### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by JimZhouZZY** [Grimmys/rpg_tactical_fantasy_game#78]
Three years passed. I think it's a good idea to publish a new release version.

**Comment by Grimmys** [comment]
I totally agree with you Jim, I'm thinking about a new release for a long time but always thought that there should be more things.

In fact, in my opinion the main issue is that level 4 is still WIP, with some bugs, and a victory condition not working properly I think.

Before next release, I think bugs need to be cleared and level 4 closed, including new gameplay mechanics introduced in this level like charge and anti-cavalry.
